packages/react-instantsearch/src/widgets/SearchBox/SearchBox.js

```diff
@@ -2,6 +2,21 @@ import React, {Component, PropTypes} from 'react';
 
 import themeable from '../../core/themeable';
 import translatable from '../../core/translatable';
+import theme from './SearchBox.css';
+
+const SubmitIcon = () =>
+  <svg xmlns="http://www.w3.org/2000/svg" id="ris-search-icon" viewBox="0 0 40 40">
```

Comment:
Ultimately having those as svg files and using some react-svg-loader (or smg) would be easier. It adds noise to the code currently. Let's keep that in mind for later. I am not sure this is easy/compatible with babel webpack loader plugin that we need to use.